release-notes/v1_66.md

```diff
@@ -191,6 +191,24 @@ The [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.pytho
 
 ![Interpreters info displayed towards right](images/1_66/active-interpreter-display.png)
 
+### Jupyter
+
+**Kernel Support**
+
+Changes were made to better support launching Conda kernels. Also, support for .env files for all platforms was added when launching kernels.
```

Comment:
I find the 'Also' statement a little weird. Maybe like this?
```markdown
Kernel startup was improved with:
- Support for more conda environments
- Support for .env files on all platforms
```

Reply:
Yeah the Also is unrelated 😆 good call. I'll change.